dialogflow/stopwatch/Stopwatch/DialogflowViewController.swift

```diff
@@ -116,11 +113,39 @@ class DialogflowViewController: UIViewController {
 
   }
 
+  @objc func presentNavigationDrawer() {
+    let bottomDrawerViewController = MDCBottomDrawerViewController()
+    bottomDrawerViewController.setTopCornersRadius(24, for: .collapsed)
+    bottomDrawerViewController.setTopCornersRadius(8, for: .expanded)
+    bottomDrawerViewController.isTopHandleHidden = false
+    bottomDrawerViewController.topHandleColor = UIColor.lightGray
+    bottomDrawerViewController.contentViewController = contentViewController
+    bottomDrawerViewController.headerViewController = headerViewController
+    bottomDrawerViewController.delegate = self
+    MDCBottomDrawerColorThemer.applySemanticColorScheme(MDCSemanticColorScheme(),
+                                                        toBottomDrawer: bottomDrawerViewController)
+    present(bottomDrawerViewController, animated: true, completion: nil)
+  }
+
+  func bottomDrawerControllerDidChangeTopInset(_ controller: MDCBottomDrawerViewController,
+                                               topInset: CGFloat) {
+    headerViewController.titleLabel.center =
+      CGPoint(x: headerViewController.view.frame.size.width / 2,
+              y: (headerViewController.view.frame.size.height + topInset) / 2)
+  }
+
   func setUpNavigationBarAndItems() {
     // AppBar Init
     self.addChildViewController(appBar.headerViewController)
     self.appBar.headerViewController.headerView.trackingScrollView = tableView
     appBar.addSubviewsToParent()
+    let barButtonLeadingItem = UIBarButtonItem()
+    barButtonLeadingItem.tintColor = ApplicationScheme.shared.colorScheme.primaryColorVariant
+    //barButtonLeadingItem.image = #imageLiteral(resourceName: "Menu")
+    barButtonLeadingItem.title = "opt"
```

Comment:
Please change this line to the following:
  barButtonLeadingItem.title = "more"